packages/core/src/context/ContextSystem.ts

```diff
@@ -13,10 +15,30 @@ let CONTEXT_UID = 0;
  */
 export class ContextSystem extends System
 {
+    public webGLVersion: number;
+    protected CONTEXT_UID: number;
+    protected gl: IRenderingContext;
+    /* eslint-disable @typescript-eslint/camelcase */
+    extensions:
```

Comment:
Access modifier?

Reply:
Yep, `getExtensions` is protected, but there are references to object from other systems, so - uknown for now.